server/protocols/jmap-draft/src/test/java/org/apache/james/jmap/draft/model/message/view/MessageFullViewTest.java

```diff
@@ -151,10 +152,19 @@ void buildShouldThrowWhenDateIsNull() {
     }
 
     @Test
-    void buildShouldThrowWhenPreviewIsNull() {
-        assertThatThrownBy(() -> MessageFullView.builder().id(TestMessageId.of(1)).blobId(BlobId.of("blobId")).threadId("threadId").fluentMailboxIds().headers(ImmutableMap.of())
-            .subject("subject").size(123).date(Instant.now()).build())
-            .isInstanceOf(IllegalStateException.class);
+    void buildNotShouldThrowWhenPreviewIsNotPresent() {
```

Comment:
s/buildNotShouldThrow/buildShouldNotThrow

Reply:
oki, it's small, I squashed